_posts/2024-10-08-cockpit-326.md

```diff
@@ -0,0 +1,76 @@
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 326, cockpit-podman 96, and cockpit-files 9:
+
+## cockpit/ws container: Connect to any Linux server
+
+The [quay.io/cockpit/ws](https://quay.io/repository/cockpit/ws)container provides Cockpit's web server in environments such as Kubernetes and [Fedora CoreOS](https://fedoraproject.org/coreos/), connecting to a remote machine via `ssh`.
+
+Now, the cockpit/ws container gained the ability to connect to _any_ Linux server, even those without Cockpit installed. This provides a convenient way to perform basic administration tasks remotely. This is similar to the [Cockpit Client flatpak](https://flathub.org/apps/org.cockpit_project.CockpitClient) ([available since release 295](https://cockpit-project.org/blog/cockpit-295.html)), but can run anywhere an OCI container can run.
+
+Currently, the following pages are available when connecting to a server without Cockpit: Overview, Metrics, Terminal, Accounts, and Networking.  More pages will be added soon, after vetting them for this use.
+
+## cockpit/ws container: Support host-specific SSH keys
+
+The [cockpit/ws](https://quay.io/repository/cockpit/ws) container now supports adding multiple SSH private keys.
+
+In addition to the existing `$COCKPIT_SSH_KEY_PATH` environment variable, you can now set host specific `$COCKPIT_SSH_KEY_PATH_{HOSTNAME}` variables, where `{HOSTNAME}` is the host name (in capital letters) used in the `Connect to:` field of the login page.
+
+_Thanks to [benniekiss](https://github.com/benniekiss) for contributing this feature!_
+
+## Bridge supports PCP; cockpit-pcp package is now obsolete
+
+The Python cockpit-bridge, introduced in Cockpit 294, now implements PCP functionality. As a result, the separate cockpit-pcp package is now obsolete.
+
+## Storage: Manage Stratis virtual filesystem sizes
+
+Cockpit supports seting a default and maximum size limit for Stratis virtual filesystems, both during creation and when making adjustments
+
+![screenshot of support for managing virtual stratis filesystem sizes]({{ site.baseurl }}/images/326-storage-support-for-managing-virtual-stratis-filesystem-sizes.png)
+
+
+## Podman: pull images from registries without search API
+
+Container creation now supports registries lacking a search API, including GitHub's container registry (ghcr.io). You can also select a specific tag for images.
+
+![screenshot of pull images from registries without search api]({{ site.baseurl }}/images/326-podman-pull-images-from-registries-without-search-api-.png)
+
+
+## Files: basic keyboard shortcuts
+
```

Comment:
Maybe at least one sentence how to get that dialog with the keyboard help?

Reply:
"From the three-dot menu next to the Upload button"? :-)

Reply:
I don't know where it is :grin: That's why I'm asking, I figure I may not be the only one.